pkg/deploy/api/types.go

```diff
@@ -441,6 +441,20 @@ type DeploymentConfigRollbackSpec struct {
 	IncludeStrategy bool
 }
 
+// DeploymentRequest is a request to a deployment config for a new deployment.
+type DeploymentRequest struct {
+	unversioned.TypeMeta
+	// Name of the deployment config for requesting a new deployment.
+	Name string
```

Comment:
if we are going to use this to instantiate the k8s deployments we might need to distinguish between DC and D here?